### ISSUE TITLE
sp.Search 500 unknown error wildcard search in people results source

### ISSUE THREAD

**Issue by c-eiser13** [pnp/pnpjs#1772]
Thank you for reporting an issue, suggesting an enhancement, or asking a question. We appreciate your feedback - to help the team understand your
needs please complete the below template to ensure we have the details to help. Thanks!

**Please check out the [Docs](https://pnp.github.io/pnpjs/) to see if your question is already addressed there. This will help us ensure our documentation covers the most frequent questions.**

### Category
- [ ] Enhancement
- [ ] Bug
- [x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [    2.5.0     ]

Please specify what version(s) of SharePoint you are targeting: [   SPO      ]

*If you are not using the latest release, please update and see if the issue is resolved before submitting an issue.*

### Expected / Desired Behavior / Question
When using Querytext: "*" against the people results source, should return all people (500) max. 

### Observed Behavior
I'm not sure if this is a bug or not, as the above works in many of our customers tenants and our own internal tenants, but I have one customer where this error is generated and we cannot seem to get around it. The call below (I've tested both sp.search and sp.searchWithCaching and have same result), results in a 500 unknown error:
![image](https://user-images.githubusercontent.com/42096464/119410337-4c938600-bcae-11eb-946b-20c6ea2450d4.png)

![image](https://user-images.githubusercontent.com/42096464/119410365-5cab6580-bcae-11eb-9253-3c8ca2613273.png)

![image](https://user-images.githubusercontent.com/42096464/119410563-aa27d280-bcae-11eb-8179-ba573cdebc63.png)

When this error was first reported last week, I was able to repro in the browser search bar, by typing an asterisk in the search bar, and an error was displayed. We opened a support ticket through the admin center since we could repro in oob search, and a week later, the support person asked us to try again, and it is now working in the browser, but still generates the error when calling the API. The support person was unable to tell us what was done, so we are not sure where the actual issue lies. We have not approached the API error yet with them as I wanted to post here first in case there is something I am missing or someone else has come across this. If I add anything else into the query in this environment, such as LastName:a*, this works, but I need to return all users.

### Steps to Reproduce
Using the latest version of @pnp/sp, make a call to search API with people results source ID and * as the query text.

**Comment by koltyakov** [comment]
Hi @c-eiser13,

Did you tried to trim down API search request to a bare minimal?

```typescript
const peopleRes = await sp.search({
  Querytext: '*',
  SourceId: 'b09a7990-05ea-4af9-81ef-edfab16c4e31'
});

console.log(peopleRes.PrimarySearchResults);
```

Does it also resulted with "An unknown error occurred"? 

Maybe something "SelectProperties" wise has been changes in search schema?

**Comment by patrick-rodgers** [comment]
If this repros through the UI then it is not a library issue, which is further supported by the fact it works in all tenants except 1. I would explore that result source and see if someone updated the configuration or other things have happened in the farm since it was working.

**Comment by c-eiser13** [comment]
@koltyakov thanks for the suggestion, I had not tried that. In doing so, it turns out it is the SortList param that is breaking the call. I'm sorting on last name like so:

![image](https://user-images.githubusercontent.com/42096464/119503486-a341a400-bd30-11eb-8238-3c261503de19.png)

If I remove this from the call, it works properly. I checked the managed property for LastName and it matches the setup for LastName in my tenant. The other interesting thing is that I can change the query text to LastName:a* and this works, even with SortList in place:

![image](https://user-images.githubusercontent.com/42096464/119504034-32e75280-bd31-11eb-81eb-63ba3de48516.png)
![image](https://user-images.githubusercontent.com/42096464/119504086-41356e80-bd31-11eb-8b03-432fca7410b9.png)

I also tried changing the property in SortList to FirstName and decreasing the page size in case there was a last name with bad data, but still generates error. I still suspect something may be awry with the index in this tenant since I was able to repro the error in the UI until yesterday, when MS support responded to our ticket and we tried again in the UI and it worked. If you have any other ideas of things to try/look at or suggest to support, I'd appreciate it. Thanks!
